airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -28,5 +28,5 @@ WORKDIR /airbyte
 ENV AIRBYTE_ENTRYPOINT "/airbyte/entrypoint.sh"
 ENTRYPOINT ["/airbyte/entrypoint.sh"]
 
-LABEL io.airbyte.version=0.1.75
+LABEL io.airbyte.version=0.1.76
```

Comment:
Same, since an image already exists we wont be able to push a new image with the same tag

Reply:
what about this?